### PR TITLE
CSS fixes: campaign wizard icons + alerts popup

### PR DESCRIPTION
**Pull request by YonatanRosemarin**
Fix CSS issues of 2 cases:
* Campaign wizard first step
* Alerts notifications pop up

## Description

Campaign - Before:
image

Campaign - After:
image

Alerts - Before:
image

Alerts - After:
image

## Motivation and Context

To fix display issue of the 2 cases described above

## How To Test This

**Need to first build CSS files from SCSS**
1. For alerts - create new alert
2. For Campaigns - go to Campaigns and click on "Create Campaign"

## Types of changes

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

### Final checklist

- [X] My code follows the code style of this project found [here](https://docs.suitecrm.com/community/contributing-code/coding-standards/).
- [ ] My change requires a change to the documentation.
- [X] I have read the [**How to Contribute**](https://docs.suitecrm.com/community/contributing-code/) guidelines.

### CODE REVIEW

File: themes/SuiteP/css/suitep-base/navbar.scss [hunk 1]
```diff
@@ -1064,7 +1064,7 @@
 }
 
 .desktop_notifications > .dropdown-menu {
-  left: auto;
+  left: -82%;
```
Comment (samus-aran) [inline themes/SuiteP/css/suitep-base/navbar.scss:1067]: This also works for mobile? I dislike having these percentages as large as that, it's almost masking the real problem I feel. If I enlarge my text on my browers (cause I often take my glasses off) will it still be in a reasonable position?

**Review by samus-aran — COMMENTED** [review]
The campaign wizard fixes look good and fairly straight forward to me :+1: 

Though I'm abit on the fence about the alerts. Should we instead look at where the anchor is of those desktop notifications rather than having to have a percentage (a very custom looking one) to position.

align_1
Is this the proposed anchor and we doing a minus from there.... Would this cause issues with mobile view? or text larger browser settings?
My thinking would it be better to move the anchor so that we are not so dependant on the right hand of the screen being in perfect alignment like so:
align_2

Hope I make sense :sweat_smile:

File: themes/SuiteP/css/suitep-base/navbar.scss (hunk 1 above)
Comment (YonatanRosemarin) [reply themes/SuiteP/css/suitep-base/navbar.scss:1067]: @samus-aran - you are correct, we shouldn't use it like that.
Do you have a suggestion on how to fix it correctly? Do you prefer I will remove it from the PR for now (I'm not sure when I will have the time to find the best solution)?

**Comment by serhiisamko091184** [comment]
Hello @YonatanRosemarin ,
thanks for your contribution, 
it's recreated due to the last comment (fix focused on the icons only): 
https://github.com/salesagility/SuiteCRM/pull/10203

Thanks! 

Regards,
Serhii

**Pull request closed without merging**